Cue/app/tabs/library/DeckView.js

```diff
@@ -92,6 +94,34 @@ class DeckView extends React.Component {
     this.props.flagCard(this.props.deckUuid, cardUuid, flag)
   }
 
+  _rateDeck = () => {
+    Alert.alert(
+      (Platform.OS === 'android'
+        ? 'Rate this deck'
+        : 'Rate This Deck'),
+      'Would you recommend this deck to others?',
```

Comment:
Yeah I'm not sure... **Recommend** and **Would Not Recommend** aren't too bad. You could also try **Yes, Would Recommend** and **No, Would Not Recommend**, though those might be too long.